mne/decoding/transformer.py

```diff
@@ -334,16 +340,17 @@ def transform(self, epochs_data, y=None):
         n_epochs, n_channels, n_times = epochs_data.shape
         X = epochs_data.reshape(n_epochs * n_channels, n_times)
 
-        psd, _ = multitaper_psd(x=X, sfreq=self.sfreq, fmin=self.fmin,
-                                fmax=self.fmax, bandwidth=self.bandwidth,
-                                adaptive=self.adaptive, low_bias=self.low_bias,
-                                n_jobs=self.n_jobs,
-                                normalization=self.normalization,
-                                verbose=self.verbose)
+        psd, freqs = multitaper_psd(x=X, sfreq=self.sfreq, fmin=self.fmin,
+                                    fmax=self.fmax, bandwidth=self.bandwidth,
+                                    adaptive=self.adaptive, low_bias=self.low_bias,
+                                    n_jobs=self.n_jobs,
+                                    normalization=self.normalization,
+                                    verbose=self.verbose)
 
         _, n_freqs = psd.shape
         psd = psd.reshape(n_epochs, n_channels, n_freqs)
-
+        self.data = psd
+        self.freqs = freqs
```

Comment:
indeed. you get psd out with the return. If you want to add the freqs attribute call it `freqs_` that would be ok for me.